### ISSUE TITLE
Spine keeps forgetting render frame range; please remember this (again?)!

### ISSUE THREAD

**Issue by bencresty** [EsotericSoftware/spine-editor#831]
I don't know at what Spine version this started, but I'm pretty sure this wasn't an issue in older versions of Spine; when exporting to an image sequence the last frame in the export dialog window always is frame 0, even after saving and re-opening the Spine file. No matter how many times we change this and save the Spine file, it keeps resetting to zero. Which is obviously not what we want and prone to issues, because it always just renders a single frame when not every time setting this end frame again on each render.

<img width="302" alt="image" src="https://github.com/user-attachments/assets/63bba539-566b-484f-8380-e35c620b64c9">

So please remember the frame range as set in the export window.

Spine Pro 4.2.36

**Comment by misaki-eymard** [comment]
The information in your report is inaccurate. If you see behavior that you think is a problem, or if you want to make a request, you can basically post it on the Spine forum: https://esotericsoftware.com/forum/
Then, if we determine that it is indeed a problem that needs to be fixed, we will create an issue ticket to see how long the problem has been occurring and the exact conditions under which it occurred.

As you mentioned, there does indeed seem to be a problem with Range's last frame setting going back to 0.

**Steps to reproduce:**
1. Create a new project, and animate the root bone.
1. Export the project in any format that allows you to specify the `Frames:Range` option with the `Range` option specified.
1. Save the Spine project, quit Spine, restart Spine and reopen the file.
1. Now you can see the last frame of the `Range` becomes 0, regardless of what the original value was.

As far as I can confirm this is reproducible even in 4.1.00. The `Range` option does not exist in 4.0, so this problem occurs regardless of the version. To reproduce this problem, Spine must be restarted. It does not happen if you open another Spine project and then reopen it.

This is indeed strange, because the`editor.json` in the `settings` folder stores the value of `rangeEnd` correctly. There may be some issue with the way the stored export settings are loaded.

**Comment by misaki-eymard** [comment]
> Looks like you yourself found out while writing your response that my report was indeed accurate. So not sure what you're trying to say here.

No, because you said "I'm pretty sure this wasn't an issue in older versions of Spine", but actually I was able to reproduce the problem even with the earliest version of the editor that allowed `Range` to be specified. Also, you wrote that the problem occurs when you reopen the Spine project, but in fact you need to restart the Spine editor itself.

When we find a problem, we try to determine when the problem started. This is because the problem may have occurred when we did an update of some kind, and seeing the changes we made at that time may allow us to fix the problem more quickly. Of course, you don't have to confirm these details, so you can start by submitting a request to investigate this type of problem by posting in the forum. We will then investigate, including the detailed conditions such as whether it is necessary to reopen the project or restart Spine to reproduce the problem, and if we find that there is indeed a problem, we will create an issue ticket. This is why I tell you to post in the forum.

**Comment by NathanSweet** [comment]
4.3.00-beta
